cloudslang_dsl_reference.rst

```diff
@@ -1407,28 +1396,6 @@ There are several types of tasks:
 -  `iterative <#iterative-task>`__
 -  `asynchronous <#asynchronous-task>`__
 
-All types of tasks may take task arguments. Task arguments may be defined using
-a standard YAML list or using the single-line syntax. When using the single-line
-syntax, all taks arguments must be `expressions <#expressions>`__.
-
-**Example - standard YAML list of task arguments**
-
-.. code-block:: yaml
-
-    - divider:
-        do:
-          divide:
-            - dividend: ${input1}
-            - divisor: "5"
-
-**Example - single-line task arguments syntax**
-
-.. code-block:: yaml
-
-    - divider:
-        do:
-          divide: dividend = ${input1}, divisor = ${"5"}
-
 Standard Task
```

Comment:
I think we should leave an example of task arguments as list. Maybe also add an argument with no default value given